actionpack/lib/action_dispatch/http/response.rb

```diff
@@ -142,7 +142,7 @@ def closed?
       private
 
         def each_chunk(&block)
-          @buf.each(&block) # extract into own method
+          @buf.each(&block) # TODO: Extract into own method.
```

Comment:
I'm not sure this is a `TODO` - looking at the original commit b43158afba7fb4de6a3530d4f4e940f0c89bd057 doesn't suggest that it is since that added `each_chunk`. Maybe just remove the comment all together.